src/components/Menu/Menu.tsx

```diff
@@ -1,24 +1,33 @@
 import { Menu as HeadlessMenu } from '@headlessui/react';
 import clsx from 'clsx';
-import React from 'react';
 import type { MouseEventHandler } from 'react';
+import { useContext, useState } from 'react';
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createPortal } from 'react-dom';
```

Comment:
Should we have `react-dom` as a regular dependency and not a devdep?

Reply:
ah yeah forgot to flag those for removal. need to revisit what the error was